Examples/HelloWorldVapor/Sources/main.swift

```diff
@@ -21,6 +21,6 @@ let app = try Application(.detect())
 app.http.server.configuration.hostname = "0.0.0.0"
 defer { app.shutdown() }
 
-app.get { _ in "Hello World, from \(myos)\n" }
+app.get { _ in "Hello World, from Vapor on \(myos)\n" }
```

Comment:
Should the text updated to say Hello World, from Hummingbird instead of Vapor?

Reply:
Thank you for your comment!   

In this branch we're adding a new Hummingbird example, but keeping the existing Vapor example.   This line adds `Vapor` to the Vapor example in `Examples/HelloWorldVapor/Sources/main.swift`, to make it easy to identify which example you're running.   The Hummingbird example is in a separate file at `Examples/HelloWorldHummingbird/Sources/main.swift`.